oidc-controller/api/core/config.py

```diff
@@ -37,7 +35,7 @@ class GlobalConfig(BaseSettings):
     DB_PASS: str = os.environ.get("OIDC_CONTROLLER_DB_USER_PWD", "oidccontrollerpass")
 
     MONGODB_URL: str = f"mongodb://{DB_USER}:{DB_PASS}@{DB_HOST}:{DB_PORT}/{DB_NAME}?retryWrites=true&w=majority"
-
+    REDISDB_URL: str = f"redis://controller-cache:6379/provider"
```

Comment:
The name/port of the cache service should be parametrized. Probably the queue name as well.